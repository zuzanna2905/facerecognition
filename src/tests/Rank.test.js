import {shallow} from 'enzyme';
import React from 'react';
import Rank from '../component/Rank/Rank';

it('renders Rank without crashing', () => {
    const initialProps = {
        name: 'John', 
        entries: 3
    }
    let  wrapper = shallow(<Rank {...initialProps}/>);
    expect(wrapper).toMatchSnapshot();
});