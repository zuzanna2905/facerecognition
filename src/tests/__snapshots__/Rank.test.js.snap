// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Rank without crashing 1`] = `
<div>
  <div
    className="white f3"
  >
    John, your current entry count is... 
  </div>
  <div
    className="white f1"
  >
    3
  </div>
</div>
`;
